packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown.tsx

```diff
@@ -128,6 +173,10 @@ export class FbFieldDropdown {
     event.stopPropagation();
 
     const objNewChoice = { value: '' };
+    if (this.level) {
```

Comment:
can we add (this.isDependentField && this.level) here

Reply:
Done.